test/sqllogictest/aggregates.slt

```diff
@@ -398,3 +398,22 @@ FROM
     v
 ----
 {"0":"c","1":"a","2":"b"}  {"0":"c","1":"a","2":"d"}  {"0":"c","1":"a","2":"d"}  {"0":"c","1":"a","2":"b"}
+
+
+# Test Reduction elision
+
+statement ok
+CREATE TABLE a (x text)
+
+statement ok
+INSERT INTO a VALUES ('a'),('b')
+
+query T
+SELECT STRING_AGG(y, ', ') FROM (SELECT x as y FROM a);
```

Comment:
This must be `STRING_AGG(y, ', ' ORDER BY y)` to be deterministic.

Reply:
Also can you remove the space after the comma in the second argument? It makes SLT ouputs weird.

Reply:
sure